fluentd/plugins/input/fluent-plugin-jfrog-siem/lib/fluent/plugin/in_jfrog_siem.rb

```diff
@@ -120,7 +120,7 @@ def run
 
             # Determine if we need to persist this record or not
             persistItem = true
-            if created_date <= last_created_date
+            if created_date < last_created_date
```

Comment:
This change will result in lost items but no duplicates. I understand why you made this change because at times there are multiple records returned at the exact same second but let's discuss it may be better to allow some duplication vs missing records.